src/lib/AutocompleteTextInput.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+  import { FormElement } from "./index.js";
+  import { v4 as uuidv4 } from "uuid";
+
+  export let label;
+  export let name;
+  export let value;
+  export let options;
+  export let dataTestId = "";
+
+  let id = `${name}-${uuidv4()}`;

```

Comment:
just `let id = uuidv4()` ought to be fine. Making the user pick an extra string that isn't displayed isn't too useful. Having meaningful element IDs could be useful if we weren't using Svelte and doing `getElementById` or something, but I've never looked at element IDs myself personally. Firefox/chrome inspector tool makes it easy enough to debug the DOM and visually see what's on the page